### DISCUSSION TITLE
Feature Request: Zigbee sensor (NAS-PD07) shows only Presence ...

### DISCUSSION THREAD

**Discussion by pelizz** [arendst/Tasmota#15648] [Ideas]
Hello,
i have some zigbee sensors 3in1 (PIR+temperature&humidity) like this https://www.zigbee2mqtt.io/devices/NAS-PD07.html.
This sensor is known working  with zigbee2mqtt but with Tasmota **show only Occupancy attribute**....:-(

This is the result of ZBstatus2 aftere pairing: {"ZbStatus2":[{"Device":"0x380A","IEEEAddr":"0xA4C138D10F7599AD","ModelId":"TS0202","Manufacturer":"_TZ3000_kmh5qpmb","Endpoints":[1],"Config":["A01.1","I01"]}]}

I tried to read manual data with commands like this (trying different cluster and attributes) but no way to have temperature and humidity: ZbSend {"Device":"0x380A","Endpoint":1, "Cluster":"0x0500","Read":"0x0001"}.
I tried also command like this:
ZbSend {"Device":"0x380A","Send":"\<cluster>!00/\<payload>"} but i do not know the right cluster and payload.

Someone can help me?

Thanks,
Maurizio

Following the weblog debug of pairing:
16:48:47.567 RSL: RESULT = {"ZbPermitJoin":"Done"}
16:48:47.582 ZIG: Something to_send, to_ack:2, to_send:2, to_end:4
16:48:47.584 ZIG: ZbEZSPSentRaw 231A000122003C
16:48:47.590 ZIG: {"ZbEZSPReceived2":"331A8001220000"}
16:48:47.591 ZIG: new ack/data received, was 2 now 3
16:48:47.593 ZIG: freeing packet 2 from memory
16:48:47.594 ZIG: ZbEZSPSentRaw 84
16:48:47.595 ZIG: {"ZbEZSPReceived":"220000"}
16:48:47.597 ZIG: Something to_send, to_ack:3, to_send:3, to_end:4
16:48:47.599 ZIG: ZbEZSPSentRaw 341B00013600FCFF00003600000000000000011E0103013C01
16:48:47.605 ZIG: {"ZbEZSPReceived2":"441B800136000034"}
16:48:47.607 ZIG: new ack/data received, was 3 now 4
16:48:47.608 ZIG: freeing packet 3 from memory
16:48:47.610 ZIG: ZbEZSPSentRaw 85
16:48:47.611 ZIG: {"ZbEZSPReceived":"36000034"}
16:48:47.614 ZIG: {"ZbEZSPReceived2":"541B90014500050000360000000000000034FF000000FFFF03013C01"}
16:48:47.615 ZIG: ZbEZSPSentRaw 86
16:48:47.617 ZIG: {"ZbEZSPReceived":"4500050000360000000000000034FF000000FFFF03013C01"}
16:48:48.579 ZIG: {"ZbEZSPReceived2":"641B90013F0006FCFF0000360000000000000034010000"}
16:48:48.581 ZIG: ZbEZSPSentRaw 87
16:48:48.582 ZIG: {"ZbEZSPReceived":"3F0006FCFF0000360000000000000034010000"}
16:49:08.226 ZIG: ZbEZSPSentRaw 80
16:49:08.227 ZIG: {"ZbEZSPReceived":"230000010A38AD99750FD138C1A404"}
16:49:08.432 ZIG: {"ZbEZSPReceived2":"041B900124000A38AD99750FD138C1A401000000"}
16:49:08.434 ZIG: ZbEZSPSentRaw 81
16:49:08.435 ZIG: {"ZbEZSPReceived":"24000A38AD99750FD138C1A401000000"}
16:49:08.438 RSL: RESULT = {"ZbState":{"Status":34,"IEEEAddr":"0xA4C138D10F7599AD","ShortAddr":"0x380A","ParentNetwork":"0x0000","JoinStatus":1,"Decision":0}}
16:49:09.526 ZIG: {"ZbEZSPReceived2":"141B90014500040000130000000000000013D4D10A38FFFF0C000A38AD99750FD138C1A48002"}
16:49:09.528 ZIG: ZbEZSPSentRaw 82
16:49:09.529 ZIG: {"ZbEZSPReceived":"4500040000130000000000000013D4D10A38FFFF0C000A38AD99750FD138C1A48002"}
16:49:09.532 RSL: RESULT = {"ZbState":{"Status":30,"IEEEAddr":"0xA4C138D10F7599AD","ShortAddr":"0x380A","PowerSource":false,"ReceiveWhenIdle":false,"Security":false}}
16:49:09.534 ZIG: ZbEZSPSend 3400000A3800000500000040010000020103020A38
16:49:09.536 ZIG: adding packet to_send, to_ack:4, to_send:4, to_end:4
16:49:09.537 ZIG: Something to_send, to_ack:4, to_send:4, to_end:5
16:49:09.539 ZIG: ZbEZSPSentRaw 421C00013400000A3800000500000040010000020103020A38
16:49:09.546 ZIG: {"ZbEZSPReceived2":"251C800134000037"}
16:49:09.547 ZIG: new ack/data received, was 4 now 5
16:49:09.549 ZIG: freeing packet 4 from memory
16:49:09.550 ZIG: ZbEZSPSentRaw 83
16:49:09.552 ZIG: {"ZbEZSPReceived":"34000037"}
16:49:09.716 ZIG: {"ZbEZSPReceived2":"351C90013F00000A380000050000004001000037010000"}
16:49:09.718 ZIG: ZbEZSPSentRaw 84
16:49:09.720 ZIG: {"ZbEZSPReceived":"3F00000A380000050000004001000037010000"}
16:49:09.723 ZIG: {"ZbEZSPReceived2":"451C90014500000000058000004001000014D4D10A38FFFF0602000A38010102"}
16:49:09.724 ZIG: ZbEZSPSentRaw 85
16:49:09.726 ZIG: {"ZbEZSPReceived":"4500000000058000004001000014D4D10A38FFFF0602000A38010102"}
16:49:09.729 RSL: RESULT = {"ZbState":{"Status":32,"ActiveEndpoints":["0x01"]}}
16:49:09.731 ZigbeeZCLSend device: 0x380A, group: 0x0000, endpoint:1, cluster:0x0000, cmd:0x00, send:"05000400"
16:49:09.733 ZIG: ZbEZSPSend 3400000A380401000001014001000001010700010005000400
16:49:09.734 ZIG: adding packet to_send, to_ack:5, to_send:5, to_end:5
16:49:09.736 ZIG: Something to_send, to_ack:5, to_send:5, to_end:6
16:49:09.737 ZIG: ZbEZSPSentRaw 551D00013400000A380401000001014001000001010700010005000400
16:49:09.745 ZIG: {"ZbEZSPReceived2":"561D800134000038"}
16:49:09.746 ZIG: new ack/data received, was 5 now 6
16:49:09.747 ZIG: freeing packet 5 from memory
16:49:09.749 ZIG: ZbEZSPSentRaw 86
16:49:09.750 ZIG: {"ZbEZSPReceived":"34000038"}
16:49:09.910 ZIG: {"ZbEZSPReceived2":"661D90013F00000A380401000001014001000038010000"}
16:49:09.912 ZIG: ZbEZSPSentRaw 87
16:49:09.913 ZIG: {"ZbEZSPReceived":"3F00000A380401000001014001000038010000"}
16:49:09.922 ZIG: {"ZbEZSPReceived2":"761D90014500000401000001014001000015D8D20A38FFFF23180101050000420654533032303204000042105F545A333030305F6B6D683571706D6202"}
16:49:09.923 ZIG: ZbEZSPSentRaw 80
16:49:09.925 ZIG: {"ZbEZSPReceived":"4500000401000001014001000015D8D20A38FFFF23180101050000420654533032303204000042105F545A333030305F6B6D683571706D6202"}
16:49:09.927 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0000","srcaddr":"0x380A","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":107,"securityuse":0,"seqnumber":21,"fc":"0x18","frametype":0,"direction":1,"disableresp":1,"manuf":"0x0000","transact":1,"cmdid":"0x01","payload":"050000420654533032303204000042105F545A333030305F6B6D683571706D62"}}
16:49:09.931 ZIG: ZbZCLRawReceived: {"0x380A":{"0000/0005":"TS0202","0000/0004":"_TZ3000_kmh5qpmb","Endpoint":1,"LinkQuality":107}}
16:49:09.934 RSL: SENSOR = {"ZbReceived":{"0x380A":{"Device":"0x380A","ModelId":"TS0202","Manufacturer":"_TZ3000_kmh5qpmb","Endpoint":1,"LinkQuality":107}}}
16:49:10.014 ZIG: {"ZbEZSPReceived2":"061D90013F0006FDFF0000130000000000000013000000"}
16:49:10.016 ZIG: ZbEZSPSentRaw 81
16:49:10.017 ZIG: {"ZbEZSPReceived":"3F0006FDFF0000130000000000000013000000"}
16:49:10.667 ZIG: {"ZbEZSPReceived2":"161D90014500000000020000004001000016D4D10A38FFFF0301000002"}
16:49:10.669 ZIG: ZbEZSPSentRaw 82
16:49:10.671 ZIG: {"ZbEZSPReceived":"4500000000020000004001000016D4D10A38FFFF0301000002"}
16:49:10.936 ZIG: {"ZbEZSPReceived2":"261D90019B00AD99750FD138C1A406"}
16:49:10.937 ZIG: ZbEZSPSentRaw 83
16:49:10.939 ZIG: {"ZbEZSPReceived":"9B00AD99750FD138C1A406"}
16:49:11.237 ZIG: ZbEZSPSend 3400000A3800000400000040010000030104030A3801
16:49:11.239 ZIG: adding packet to_send, to_ack:6, to_send:6, to_end:6
16:49:11.287 ZIG: Something to_send, to_ack:6, to_send:6, to_end:7
16:49:11.289 ZIG: ZbEZSPSentRaw 631E00013400000A3800000400000040010000030104030A3801
16:49:11.296 ZIG: {"ZbEZSPReceived2":"371E80013400003C"}
16:49:11.297 ZIG: new ack/data received, was 6 now 7
16:49:11.299 ZIG: freeing packet 6 from memory
16:49:11.300 ZIG: ZbEZSPSentRaw 84
16:49:11.301 ZIG: {"ZbEZSPReceived":"3400003C"}
16:49:11.649 ZIG: {"ZbEZSPReceived2":"471E90014500040000360000000000000019CCCF0A38FFFF0302B40102"}
16:49:11.651 ZIG: ZbEZSPSentRaw 85
16:49:11.652 ZIG: {"ZbEZSPReceived":"4500040000360000000000000019CCCF0A38FFFF0302B40102"}
16:49:11.680 ZIG: {"ZbEZSPReceived2":"571E90013F00000A38000004000000400100003C010000"}
16:49:11.682 ZIG: ZbEZSPSentRaw 86
16:49:11.683 ZIG: {"ZbEZSPReceived":"3F00000A38000004000000400100003C010000"}
16:49:11.690 ZIG: {"ZbEZSPReceived2":"671E9001450000000004800000400100001AC8CE0A38FFFF1D03000A3818010401020401040100000503000000040010060019000A0002"}
16:49:11.692 ZIG: ZbEZSPSentRaw 87
16:49:11.694 ZIG: {"ZbEZSPReceived":"450000000004800000400100001AC8CE0A38FFFF1D03000A3818010401020401040100000503000000040010060019000A0002"}
16:49:11.697 RSL: RESULT = {"ZbState":{"Status":33,"Device":"0x380A","Endpoint":"0x01","ProfileId":"0x0104","DeviceId":"0x0402","DeviceVersion":1,"InClusters":["0x0001","0x0500","0x0003","0x0000"],"OutClusters":["0x1000","0x0006","0x0019","0x000A"]}}
16:49:11.956 ZIG: Zigbee Devices Data saved in File System (40 bytes)
16:49:12.102 ZIG: {"ZbEZSPReceived2":"771E90013F0006FCFF0000360000000000000019000000"}
16:49:12.104 ZIG: ZbEZSPSentRaw 80
16:49:12.105 ZIG: {"ZbEZSPReceived":"3F0006FCFF0000360000000000000019000000"}
16:49:13.707 ZigbeeZCLSend device: 0x380A, group: 0x0000, endpoint:1, cluster:0x0500, cmd:0x00, send:"0100"
16:49:13.709 ZIG: ZbEZSPSend 3400000A38040100050101400100000201050002000100
16:49:13.710 ZIG: adding packet to_send, to_ack:7, to_send:7, to_end:7
16:49:13.757 ZIG: Something to_send, to_ack:7, to_send:7, to_end:0
16:49:13.759 ZIG: ZbEZSPSentRaw 701F00013400000A38040100050101400100000201050002000100
16:49:13.766 ZIG: {"ZbEZSPReceived2":"001F80013400003D"}
16:49:13.768 ZIG: new ack/data received, was 7 now 0
16:49:13.769 ZIG: freeing packet 7 from memory
16:49:13.770 ZIG: ZbEZSPSentRaw 81
16:49:13.772 ZIG: {"ZbEZSPReceived":"3400003D"}
16:49:13.951 ZIG: {"ZbEZSPReceived2":"101F90013F00000A38040100050101400100003D010000"}
16:49:13.952 ZIG: ZbEZSPSentRaw 82
16:49:13.954 ZIG: {"ZbEZSPReceived":"3F00000A38040100050101400100003D010000"}
16:49:13.958 ZIG: {"ZbEZSPReceived2":"201F9001450000040100050101400100001BDCD30A38FFFF09180201010000310D0002"}
16:49:13.960 ZIG: ZbEZSPSentRaw 83
16:49:13.961 ZIG: {"ZbEZSPReceived":"450000040100050101400100001BDCD30A38FFFF09180201010000310D0002"}
16:49:13.963 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0500","srcaddr":"0x380A","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":110,"securityuse":0,"seqnumber":27,"fc":"0x18","frametype":0,"direction":1,"disableresp":1,"manuf":"0x0000","transact":2,"cmdid":"0x01","payload":"010000310D00"}}
16:49:13.965 ZIG: ZbZCLRawReceived: {"0x380A":{"0500/0001":13,"Endpoint":1,"LinkQuality":110}}
16:49:14.361 RSL: SENSOR = {"ZbReceived":{"0x380A":{"Device":"0x380A","ZoneType":13,"Endpoint":1,"LinkQuality":110}}}
16:49:15.710 ZigbeeZCLSend device: 0x380A, group: 0x0000, endpoint:1, cluster:0x0500, cmd:0x02, send:"1000F067F0BFFEFF843C00"
16:49:15.712 ZIG: ZbEZSPSend 3400000A380401000501014001000003010E0003021000F067F0BFFEFF843C00
16:49:15.713 ZIG: adding packet to_send, to_ack:0, to_send:0, to_end:0
16:49:15.762 ZIG: Something to_send, to_ack:0, to_send:0, to_end:1
16:49:15.764 ZIG: ZbEZSPSentRaw 032000013400000A380401000501014001000003010E0003021000F067F0BFFEFF843C00
16:49:15.773 ZIG: {"ZbEZSPReceived2":"312080013400003E"}
16:49:15.775 ZIG: new ack/data received, was 0 now 1
16:49:15.776 ZIG: freeing packet 0 from memory
16:49:15.777 ZIG: ZbEZSPSentRaw 84
16:49:15.779 ZIG: {"ZbEZSPReceived":"3400003E"}
16:49:16.073 ZIG: Zigbee Devices Data saved in File System (41 bytes)
16:49:16.100 ZIG: {"ZbEZSPReceived2":"412090013F00000A38040100050101400100003E010000"}
16:49:16.102 ZIG: ZbEZSPSentRaw 85
16:49:16.103 ZIG: {"ZbEZSPReceived":"3F00000A38040100050101400100003E010000"}
16:49:16.106 ZIG: {"ZbEZSPReceived2":"51209001450000040100050101000100001FD0D00A38FFFF041803040002"}
16:49:16.108 ZIG: ZbEZSPSentRaw 86
16:49:16.109 ZIG: {"ZbEZSPReceived":"450000040100050101000100001FD0D00A38FFFF041803040002"}
16:49:16.111 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0500","srcaddr":"0x380A","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":102,"securityuse":0,"seqnumber":31,"fc":"0x18","frametype":0,"direction":1,"disableresp":1,"manuf":"0x0000","transact":3,"cmdid":"0x04","payload":"00"}}
16:49:16.115 RSL: RESULT = {"ZbResponse":{"Device":"0x380A","Command":"0500_04","Status":0,"StatusMessage":"SUCCESS","Endpoint":1,"LinkQuality":102}}
16:49:16.117 ZIG: ZbZCLRawReceived: {"0x380A":{"Endpoint":1,"LinkQuality":102}}
16:49:17.335 ZIG: {"ZbEZSPReceived2":"612090014500000401000501010001000020D4D10A38FFFF0909AF00010000FF000002"}
16:49:17.336 ZIG: ZbEZSPSentRaw 87
16:49:17.338 ZIG: {"ZbEZSPReceived":"4500000401000501010001000020D4D10A38FFFF0909AF00010000FF000002"}
16:49:17.340 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0500","srcaddr":"0x380A","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":105,"securityuse":0,"seqnumber":32,"fc":"0x09","frametype":1,"direction":1,"disableresp":0,"manuf":"0x0000","transact":175,"cmdid":"0x00","payload":"010000FF0000"}}
16:49:17.343 ZigbeeZCLSend device: 0x380A, group: 0x0000, endpoint:1, cluster:0x0500, cmd:0x0B, send:"0000"
16:49:17.344 ZIG: ZbEZSPSend 3400000A3804010005010100000000AF010510AF0B0000
16:49:17.345 ZIG: adding packet to_send, to_ack:1, to_send:1, to_end:1
16:49:17.348 ZIG: ZbZCLRawReceived: {"0x380A":{"0500<00":"010000FF0000","ZoneStatusChange":1,"0406/0000":1,"Endpoint":1,"LinkQuality":105}}
16:49:17.352 RSL: SENSOR = {"ZbReceived":{"0x380A":{"Device":"0x380A","0500<00":"010000FF0000","ZoneStatusChange":1,"Occupancy":1,"Endpoint":1,"LinkQuality":105}}}
16:49:17.354 ZIG: Something to_send, to_ack:1, to_send:1, to_end:2
16:49:17.356 ZIG: ZbEZSPSentRaw 172100013400000A3804010005010100000000AF010510AF0B0000
16:49:17.363 ZIG: {"ZbEZSPReceived2":"722180013400003F"}
16:49:17.364 ZIG: new ack/data received, was 1 now 2
16:49:17.365 ZIG: freeing packet 1 from memory
16:49:17.367 ZIG: ZbEZSPSentRaw 80
16:49:17.368 ZIG: {"ZbEZSPReceived":"3400003F"}
16:49:26.213 ZIG: ZbEZSPSentRaw 81
16:49:26.214 ZIG: {"ZbEZSPReceived":"3F00000A38040101000101400100004A010000"}
16:49:26.218 ZIG: {"ZbEZSPReceived2":"152C9001450000040101000101000100002BD8D20A38FFFF0B180507000020000000210002"}
16:49:26.220 ZIG: ZbEZSPSentRaw 82
16:49:26.221 ZIG: {"ZbEZSPReceived":"450000040101000101000100002BD8D20A38FFFF0B180507000020000000210002"}
16:49:26.223 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0001","srcaddr":"0x380A","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":107,"securityuse":0,"seqnumber":43,"fc":"0x18","frametype":0,"direction":1,"disableresp":1,"manuf":"0x0000","transact":5,"cmdid":"0x07","payload":"0000200000002100"}}
16:49:26.227 ZIG: ZbZCLRawReceived: {"0x380A":{"ConfigResponse":{"BatteryVoltage":{"Status":0,"StatusMsg":"SUCCESS"},"BatteryPercentage":{"Status":0,"StatusMsg":"SUCCESS"}},"Endpoint":1,"LinkQuality":107}}
16:49:26.230 RSL: SENSOR = {"ZbReceived":{"0x380A":{"Device":"0x380A","ConfigResponse":{"BatteryVoltage":{"Status":0,"StatusMsg":"SUCCESS"},"BatteryPercentage":{"Status":0,"StatusMsg":"SUCCESS"}},"Endpoint":1,"LinkQuality":107}}}
16:49:26.519 WIF: Checking connection...
16:49:27.722 ZIG: auto-bind `ZbSend {"Device":"0x380A","Config":{"ZoneStatus":{"MinInterval":1,"MaxInterval":3600}}}`
16:49:27.724 ZigbeeZCLSend device: 0x380A, group: 0x0000, endpoint:1, cluster:0x0500, cmd:0x06, send:"000200190100100E"
16:49:27.725 ZIG: ZbEZSPSend 3400000A380401000501014001000006010B100606000200190100100E
16:49:27.727 ZIG: adding packet to_send, to_ack:5, to_send:5, to_end:5
16:49:27.770 ZIG: Something to_send, to_ack:5, to_send:5, to_end:6
16:49:27.772 ZIG: ZbEZSPSentRaw 522D00013400000A380401000501014001000006010B100606000200190100100E
16:49:27.780 ZIG: {"ZbEZSPReceived2":"262D80013400004B"}
16:49:27.781 ZIG: new ack/data received, was 5 now 6
16:49:27.783 ZIG: freeing packet 5 from memory
16:49:27.784 ZIG: ZbEZSPSentRaw 83
16:49:27.785 ZIG: {"ZbEZSPReceived":"3400004B"}
16:49:28.047 ZIG: {"ZbEZSPReceived2":"362D90013F00000A38040100050101400100004B010000"}
16:49:28.048 ZIG: ZbEZSPSentRaw 84
16:49:28.050 ZIG: {"ZbEZSPReceived":"3F00000A38040100050101400100004B010000"}
16:49:28.053 ZIG: {"ZbEZSPReceived2":"462D9001450000040100050101000100002CD8D20A38FFFF071806070000020002"}
16:49:28.055 ZIG: ZbEZSPSentRaw 85
16:49:28.057 ZIG: {"ZbEZSPReceived":"450000040100050101000100002CD8D20A38FFFF071806070000020002"}
16:49:28.059 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0500","srcaddr":"0x380A","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":107,"securityuse":0,"seqnumber":44,"fc":"0x18","frametype":0,"direction":1,"disableresp":1,"manuf":"0x0000","transact":6,"cmdid":"0x07","payload":"00000200"}}
16:49:28.063 ZIG: ZbZCLRawReceived: {"0x380A":{"ConfigResponse":{"ZoneStatus":{"Status":0,"StatusMsg":"SUCCESS"}},"Endpoint":1,"LinkQuality":107}}
16:49:28.066 RSL: SENSOR = {"ZbReceived":{"0x380A":{"Device":"0x380A","ConfigResponse":{"ZoneStatus":{"Status":0,"StatusMsg":"SUCCESS"}},"Endpoint":1,"LinkQuality":107}}}
16:49:28.162 ZIG: {"ZbEZSPReceived2":"562D9001450000040100050101400100002DD8D20A38FFFF0818B40A020019010002"}
16:49:28.164 ZIG: ZbEZSPSentRaw 86
16:49:28.165 ZIG: {"ZbEZSPReceived":"450000040100050101400100002DD8D20A38FFFF0818B40A020019010002"}
16:49:28.167 ZIG: {"ZbZCLReceived":{"groupid":0,"clusterid":"0x0500","srcaddr":"0x380A","srcendpoint":1,"dstendpoint":1,"wasbroadcast":0,"LinkQuality":107,"securityuse":0,"seqnumber":45,"fc":"0x18","frametype":0,"direction":1,"disableresp":1,"manuf":"0x0000","transact":180,"cmdid":"0x0A","payload":"0200190100"}}
16:49:28.170 ZIG: ZbZCLRawReceived: {"0x380A":{"0500/0002":1,"Endpoint":1,"LinkQuality":107}}
16:49:28.563 RSL: SENSOR = {"ZbReceived":{"0x380A":{"Device":"0x380A","ZoneStatus":1,"Occupancy":1,"Endpoint":1,"LinkQuality":107}}}
16:49:46.540 WIF: Checking connection...
16:49:47.595 RSL: RESULT = {"ZbState":{"Status":20,"Message":"Pairing mode disabled"}}

**Comment by Jason2866** [comment]
Probably TUYA "enhanced" the zigbee protocol (again) with there own ideas.
To support Tuya devices it is often needed to add device specific code, since Tuya does not care about zigbee standards. Since Tasmota has limited resources, not every device specific code can be added.

**Comment by s-hadinger** [comment]
I will have a look later. Still busy with the Sonoff Zigbee Pro

**Reply by pelizz** [reply]
Take your time... Thanks.

**Reply by pelizz** [reply]
sorry for the insistence .... is there any indication for me? Thanks
